sys/color/color.c

```diff
@@ -175,3 +176,27 @@ void color_rgb2str(const color_rgb_t *rgb, char* str)
     tmp = rgb->b & 0x0F;
     str[5] = (tmp > 9) ? ('A' - 10 + tmp) : ('0' + tmp);
 }
+
+void color_rgb_invert(const color_rgb_t *rgb, color_rgb_t *inv_rgb)
+{
+    inv_rgb->r = 255 - rgb->r;
+    inv_rgb->g = 255 - rgb->g;
+    inv_rgb->b = 255 - rgb->b;
+}
```

Comment:
This might be more ROM efficient, when inlined... (should be easy to test this, too ;-P)